src/net/uds/listener.rs

```diff
@@ -0,0 +1,98 @@
+use super::UnixStream;
+use crate::event::Source;
+use crate::unix::SocketAddr;
+use crate::unix::SourceFd;
+use crate::{sys, Interests, Registry, Token};
+
+use std::io;
+use std::os::unix::io::{AsRawFd, FromRawFd, IntoRawFd, RawFd};
+use std::os::unix::net;
+use std::path::Path;
+
+/// A non-blocking Unix domain socket server.
+#[derive(Debug)]
+pub struct UnixListener {
+    std: std::os::unix::net::UnixListener,
```

Comment:
Same as for `UnixDatagram`:
* `std` -> `sys`,
* `std::os::unix::net::UnixListener` -> `net::UnixListener`, and
* add `SelectorId`.

Reply:
Answered below in a PR comment